articles/article-088-no-free-lunch.md

```diff
@@ -0,0 +1,25 @@
+<!-- BEGIN TITLE -->
+# There is no such thing as a free lunch
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+In Computer Science we have  a wide range of very different problems and accordingly many approaches to tackle them.
+We might for example try to find some 'optimal' solution for a search problem, but it is unclear which specific algorithm one should use.
```

Comment:
We might try to find an optimal solution for a search problem, but it is unclear which specific algorithm to use. 
- you can par down many words to make it more succinct 
- the optimal in quotes make it seem that they are not trying to find the true optimal solution, but in most cases that is what people want to do (unless you are implying that there is no real optimal solution in which case there is no solution or you have an undecidable problem)

Reply:
That was a remainder from my Machine Learning version, where in my opinion the optimality of a solution strongly depends on what you are looking for and what assumptions you make.

But yes I agree that if limited to search problems I can remove that.

Reply:
removed quotes and 'specific', but I don't want to remove too much